### PR TITLE
DOC: sponsor notice

### PR DESCRIPTION
**Pull request by jreback**
cc @jorisvandenbossche 
cc @cpcloud 
cc @shoyer 
cc @wesm

### CODE REVIEW

File: doc/source/whatsnew/v0.17.1.txt [hunk 1]
```diff
@@ -3,6 +3,10 @@
 v0.17.1 (November 21, 2015)
 ---------------------------
 
+.. note::
+
+   We are proud to announce that *pandas* has become a sponsored project of the not-for-profit [NUMFocus organization](http://numfocus.org/news/2015/10/09/numfocus-announces-new-fiscally-sponsored-project-pandas/). This will help ensure the success of development of *pandas* as a world-class open-source project.
```
Comment (TomAugspurger) [inline doc/source/whatsnew/v0.17.1.txt]: This will need to be a restructured text link, not markdown.

**Comment by shoyer** [comment]
Seems like a good idea to me

**Comment by jreback** [comment]
@jorisvandenbossche @TomAugspurger 

anyone see what's wrong with this link? at the top
http://pandas-docs.github.io/pandas-docs-travis/whatsnew.html

**Comment by TomAugspurger** [comment]
`:: _numfocus organization:` should be
`.. _numfocus organization:` I think.

**Comment by jorisvandenbossche** [comment]
@jreback I already fixed it, but it is still in the build queue I suppose, so it is not yet fixed online

**Comment by jreback** [comment]
@jorisvandenbossche oh, I just pushed the fix :)

**Comment by jorisvandenbossche** [comment]
but in the wrong direction! :-) (you made it incorrect again)

**Comment by jreback** [comment]
@jorisvandenbossche oh really? shoot. ok how about your turn :)

**Comment by jorisvandenbossche** [comment]
Reverting on master is bad practice I suppose?

**Comment by jreback** [comment]
no it's fine go ahead

**Pull request merged**